app/models/custom_field.rb

```diff
@@ -129,7 +129,7 @@ def value_types
 
     if has_embeddable_media
       if result
-        result << :embeddable_media
+        result << :url
       else
         raise "has_embeddable_media flag enabled for unexpected data_type: #{data_type}"
       end
```

Comment:
What do you think about a project-wide replace `embeddable_media` with `linked_doc` or something like that?